tempest/api/workloadmgr/snapshot/test_snapshot_mount.py

```diff
@@ -93,3 +93,137 @@ def test_1_filesearch_default_parameters(self):
             reporting.set_test_script_status(tvaultconf.FAIL)
             reporting.test_case_to_write()
+
+
+    @test.attr(type='smoke')
+    @test.idempotent_id('28fd0710-ef00-42b0-93f0-9dbb3ffc5bee')
+    def test_2_umount_snapshot(self):
+        reporting.add_test_script(str(__name__) + "_umount_snapshot")
+        try:
+            if self.exception != "":
+                LOG.debug("pre req failed")
+                reporting.add_test_step(str(self.exception), tvaultconf.FAIL)
+                raise Exception (str(self.exception))
+            LOG.debug("pre req completed")
+
+            global instances_ids
+            global snapshot_ids
+            global wid
+            global security_group_id
+            global volumes_ids
+            global fvm_id
+            global floating_ips_list
+            instances_ids = self.instances_ids
+            snapshot_ids = self.snapshot_ids
+            wid = self.wid
+            volumes_ids = self.volumes_ids
+            security_group_id = self.security_group_id
+            fvm_id = self.fvm_id
+            dismount_snapshot_id = snapshot_ids[0]
+            floating_ips_list = self.floating_ips_list
+
+            LOG.debug("dismount snasphot of a  snapshot")
+            is_dismounted = self.dismount_snapshot(dismount_snapshot_id)
+            LOG.debug("VALUE OF is_dismounted: " + str(is_dismounted))
+            if is_dismounted == True:
+                LOG.debug("dismount snapshot with full snapshot is  successful")
+                reporting.add_test_step("Verification of dismount snapshot with full snapshot", tvaultconf.PASS)
+                reporting.test_case_to_write()
+            else:
+                LOG.debug("dismount snapshot with full snapshot is unsuccessful")
+                reporting.add_test_step("Verification of dismount snapshot with full snapshot", tvaultconf.FAIL)
+                raise Exception ("Snapshot dismount with full_snapshot  does not execute correctly")
+            
+            LOG.debug("validate that snapshot is umounted from FVM")
+            self.ssh = self.SshRemoteMachineConnectionWithRSAKey(str(floating_ips_list[1]))
+            output_list = self.validate_snapshot_mount(self.ssh)
+            self.ssh.close()
+            
+            for i in output_list:
+                if '/home/ubuntu/mount_data_b' not in i:
+                   LOG.debug("connect to fvm and check mountpoint is unmounted from FVM instance")
+                   reporting.add_test_step("Verify that mountpoint is unmounted from FVM instance", tvaultconf.PASS)
+                   reporting.test_case_to_write()
+                else:
+                   LOG.debug("mount snapshot with full snapshot is unsuccessful on FVM")
+                   reporting.add_test_step("Verify that mountpoint is unmounted from FVM instance", tvaultconf.FAIL)
+                   raise Exception ("mountpoint is not showing on FVM instance")
+                if 'File_1.txt' not in i:
```

Comment:
This condition is not required, as disk is not mounted then no need to look for files.